server/sonar-server/src/main/java/org/sonar/server/organization/OrganizationFeature.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.organization;
+
+import org.sonar.db.DbSession;
+
+public interface OrganizationFeature {
```

Comment:
The api interface `Server` is not a good candidate. Currently there's no need to share the flag with scanner environment. I don't see alternatives right now on server/CE sides. Still I consider that as a shortcut that could be added later (to an alternative to `Server`). Adding dependencies to `OrganizationFeature` where needed is not a problem for now.

Meanwhile renaming `OrganizationFeature` is fine for me (I'm +0). As you said `OrganizationSupport` can be misleading. I suggest`OrganizationFlags` or `OrganizationSettings`, then adding `allowAnyoneToCreate()` and `enablePersonalOrgs()` could be great.